src/App.tsx

```diff
@@ -1,156 +1,90 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { createTodo, getTodos } from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { Notification } from './components/Notification/Notification';
+import { FilterType } from './types/FiterType';
 
-const USER_ID = 0;
+const USER_ID = 11085;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<FilterType>('all');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const fetchTodos = async () => {
+    try {
+      const fetchedTodos = await getTodos(USER_ID);
+
+      setTodos(fetchedTodos);
+    } catch (error) {
+      setErrorMessage('Unable to fetch todos.');
+    }
+  };
```

Comment:
you can declare this function inside the `useEffect` to avoid redeclaring on each render